recipes/fragpipe/meta.yaml

```diff
@@ -0,0 +1,64 @@
+{% set name = "FragPipe" %}
+{% set version = "20.0" %}
+{% set philosopher_version = "5.1.0" %}
+
+# These keys were generated ONLY for the testing of this bioconda package.
+# Users must generate their own key by agreeing to the terms at https://msfragger-upgrader.nesvilab.org/upgrader/ and https://msfragger.arsci.com/ionquant/.
+{% set msfragger_academic_use_only_key = "f28c4be2-759b0a11-6c1ea652-1240565a" %}
+{% set ionquant_academic_use_only_key = "47cb2953-b4bf7726-2c96f5fe-8f1013aa" %}
+
+package:
+  name: fragpipe
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+  run_exports:
+    - {{ pin_subpackage('fragpipe', max_pin="x.x") }}
+
+source:
+  - url: https://github.com/Nesvilab/{{ name }}/releases/download/{{ version }}/{{ name }}-{{ version }}.zip
+    sha256: 0d77db85ae26c190a915823e1c68238f3a19533810734b008564b2a8d85d1bd6
+    folder: fragpipe_source
+  - url: https://github.com/Nesvilab/philosopher/releases/download/v{{ philosopher_version }}/philosopher_v{{ philosopher_version }}_linux_amd64.zip
+    sha256: cf15a2d6321ecb530170b2410f5873f89e283b52d5f1ab0f27eebc124f4f4a1a
+    folder: philosopher_source
+
+requirements:
+  run:
+    - openjdk =11
```

Comment:
do you need to strict pins here?

Reply:
Python needs to be 3.9, otherwise I've changed the others to `>=`.